buildstockbatch/gcp/gcp.py

```diff
@@ -16,13 +16,17 @@
 :license: BSD-3
 """
 import argparse
+from datetime import datetime
 import docker
 import json
 import logging
 import os
 import pathlib
 import re
 
+from google.cloud import batch_v1
```

Comment:
I'm not sure what the convention is, but maybe this should be within the rest of the imports (rather than its own dedicated block)?

Reply:
I don't think they've been separating out 3rd-party commits ([their aws.py](https://github.com/NREL/buildstockbatch/blob/develop/buildstockbatch/aws/aws.py), for example)